src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -55,6 +58,10 @@ public void init(FeedbackSessionAttributes feedbackSession, List<FeedbackQuestio
                                       instructor.courseId, question, i + 1);
         }
 
+        this.courseDetails = courseDetails;
+        // numOfInstructors can be different from instructorList.size()
```

Comment:
Your comment here and [your code](#diff-590458d6aeb0be2612217f008cc3818cR71) does not tally.